### ISSUE TITLE
Not starting after hammerspoon 0.9.80

### ISSUE THREAD

**Issue by mattiaslundberg** [agzam/spacehammer#71]
After updating to hammerspoon 0.9.80 spacehammer fails to load, have tried both with my configuration and the default one. I get the following error message:
```
2020-09-22 13:54:37: -- Loading ~/.hammerspoon/init.lua
2020-09-22 13:54:37: -- Loading extension: alert
2020-09-22 13:54:37: *** ERROR: ...app/Contents/Resources/extensions/hs/_coresetup/init.lua:651: module 'fennel' not found:
	no field package.preload['fennel']
	no file '/Users/mattias/.hammerspoon/fennel.lua'
	no file '/Users/mattias/.hammerspoon/fennel/init.lua'
	no file '/Users/mattias/.hammerspoon/Spoons/fennel.spoon/init.lua'
	no file '/usr/local/share/lua/5.4/fennel.lua'
	no file '/usr/local/share/lua/5.4/fennel/init.lua'
	no file '/usr/local/lib/lua/5.4/fennel.lua'
	no file '/usr/local/lib/lua/5.4/fennel/init.lua'
	no file './fennel.lua'
	no file './fennel/init.lua'
	no file '/Applications/Hammerspoon.app/Contents/Resources/extensions/fennel.lua'
	no file '/Applications/Hammerspoon.app/Contents/Resources/extensions/fennel/init.lua'
	no file '/Users/mattias/.luarocks/share/lua/5.4/fennel.lua'
	no file '/Users/mattias/.luarocks/share/lua/5.4/fennel/init.lua'
	no file '/Users/mattias/.luarocks/share/lua/5.3/fennel.lua'
	no file '/Users/mattias/.luarocks/share/lua/5.3/fennel/init.lua'
	no file '/Users/mattias/.hammerspoon/fennel.so'
	no file '/usr/local/lib/lua/5.4/fennel.so'
	no file '/usr/local/lib/lua/5.4/loadall.so'
	no file './fennel.so'
	no file '/Applications/Hammerspoon.app/Contents/Resources/extensions/fennel.so'
	no file '/Users/mattias/.luarocks/lib/lua/5.4/fennel.so'
	no file '/Users/mattias/.luarocks/lib/lua/5.3/fennel.so'
stack traceback:
	[C]: in function 'rawrequire'
	...app/Contents/Resources/extensions/hs/_coresetup/init.lua:651: in function 'require'
	/Users/mattias/.hammerspoon/init.lua:22: in main chunk
	[C]: in function 'xpcall'
	...app/Contents/Resources/extensions/hs/_coresetup/init.lua:702: in function 'hs._coresetup.setup'
	(...tail calls...)
```

Downgrading to 0.9.78 solves the issue. Have tried to reinstall luarocks and fennel with brew but that doesn't help.

I'm not sure if this is something that's broken on my machine or if it's a bug somewhere, any assistance to figure this out is appreciated!

**Comment by eccentric-j** [comment]
This as reported in #68 and resolved in #70 which includes instructions on how to fix it. The pull request covers how to fix it and an alternate version can be found https://github.com/Hammerspoon/hammerspoon/wiki/Luarocks-and-Hammerspoon.

**Comment by mattiaslundberg** [comment]
Thank you, worked perfectly!
